cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx

```diff
@@ -245,11 +245,12 @@ function TagAnnotationSidebar(props: StateToProps & DispatchToProps): JSX.Elemen
                         </Checkbox>
                     </Col>
                 </Row>
-                <Row type='flex' justify='start'>
+                <Row type='flex' justify='start' className='cvat-tag-anntation-sidebar-frame-tags'>
```

Comment:
Probably typos: cvat-tag-annotation-sidebar-frame-tags

Reply:
Typo fixed.
These taken from [another element](https://github.com/openvinotoolkit/cvat/blob/develop/cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/tag-annotation-sidebar.tsx#L236)  whith contains typo. Do I need to fix this?

Reply:
Typo "cvat-tag-anntation-sidebar-checkbox-skip-frame" fixed on "cvat-tag-annotation-sidebar-checkbox-skip-frame".